File: Javascript/3-Basic-Actions/6-Get-Attribute.js
const { Builder, By } = require('selenium-webdriver');

(async function(){
    let driver = await new Builder().forBrowser('chrome').build();
    try{
        await driver.get('https://www.python.org/');
        /**
         * Fetch the DOM node
         * <button type="submit" name="submit" id="submit" class="search-button" title="Submit this Search" tabindex="3">
         * GO
         * </button>
         */
        const my_element = await driver.findElement(By.id('submit'));
        const my_element_class = await my_element.getAttribute('class');
        console.log(my_element_class);
        
        const my_element_title = await my_element.getAttribute('title');
        console.log(my_element_title);
    } finally {
        driver.quit()
    }
})()

File: Python/3-Basic-Actions/6-Get-Attribute.py
from selenium import webdriver;

driver = webdriver.Chrome()

try:
    driver.get('https://www.python.org/')
    '''
    Fetch the DOM node
    <button type="submit" name="submit" id="submit" class="search-button" title="Submit this Search" tabindex="3">
        GO
    </button>
    '''
    my_element = driver.find_element('id', 'submit')

    my_element_class = my_element.get_attribute('class')        # value of class attribute in DOM node
    print("My Element class is '%s'" % my_element_class)

    my_element_title = my_element.get_attribute('title')        # value of title attribute in DOM node
    print("My Element title is '%s'" % my_element_title)


finally:
    driver.quit()